src/ifcblenderexport/blenderbim/bim/prop.py

```diff
@@ -1343,14 +1343,6 @@ class BIMProperties(PropertyGroup):
         items=[("NONE", "None", ""), ("WHITELIST", "Whitelist", ""), ("BLACKLIST", "Blacklist", ""),],
         name="Import Filter",
     )
-    ifc_selector: StringProperty(default="", name="IFC Selector")
```

Comment:
Don't delete this `ifc_selector` property just yet... since it is still used by the MVD UI. Later on when the MVD UI is refactored, it may be deleted.